File: 프로그래머스/lv2/42584. 주식가격/주식가격.py
def solution(prices):
    answer = []
    
    for i in range(len(prices)-1):
        time = 0
        for j in range(i+1,len(prices)):
            if prices[i] > prices[j]:
                time += 1
                break
            else:
                time += 1
        answer.append(time)
    answer.append(0)
    return answer

File: 프로그래머스/2/42584. 주식가격/주식가격.java
class Solution {
    public int[] solution(int[] prices) {
        int[] answer = new int[prices.length];
        
        for (int i = 0;i < answer.length; i++) {
            answer[i] = 0;
            for (int j = i + 1; j<answer.length; j++) {
                if (prices[i] > prices[j]) {
                    answer[i]++;
                    break;
                } else {
                    answer[i]++;
                }
            }
        }
        
        return answer;
    }
}